client/post-editor/style.scss

```diff
@@ -1,3 +1,49 @@
+.dialog.confirmation-dialog {
+	max-width: 450px;
+	border-radius: 3px;
+
+	h1 {
```

Comment:
We suggest again tag name selectors in favor of classname using `.component__element` structure. This also helps avoid the need for nesting.

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md